backend/organization/models/status.py

```diff
@@ -70,3 +70,94 @@ class Meta:
 
     def __str__(self):
         return "Project status  %s" % (self.name)
+
+
+def project_type_image_path(instance, filename):
+    return "project_type_icons/{}/{}".format(instance.id, filename)
+
+
+class ProjectTypes(models.Model):
+    name = models.CharField(
+        help_text="Name of the project type",
+        verbose_name="Name",
+        max_length=512,
+        null=True,
+        blank=True,
+    )
```

Comment:
What does this represent here? Can you elaborate?

Reply:
Are you referencing the null=True and blank=True? I just took the standard settings but I agree that we should make those mandatory.